bundles/Expense/Resources/views/expense/new.html.twig

```diff
@@ -1,7 +1,15 @@
 {% extends 'base.html.twig' %}
 
 {% block content %}
-    {{ form(form) }}
+    {{ form_start(form) }}
+        {{ form_row(form.project) }}
+        {{ form_row(form.issue_key) }}
+        {{ form_row(form.category) }}
+        <div class="text-right">
+            <a href="{{ path('expense_admin_category_index') }}">{{ 'expense.category_admin'|trans }}</a>
+        </div>
+        {{ form_rest(form) }}
```

Comment:
Strictly speaking, this is not needed (`form_end` will render any not yet rendered fields), but being explicit never hurts.

Reply:
I know, but it signals that there are more pieces to the form.